python/ray/data/datasource/torch_datasource.py

```diff
@@ -0,0 +1,62 @@
+from typing import Callable, List
+import logging
+
+import torch.utils.data
+
+from ray.data.block import Block, BlockMetadata, T
+from ray.data.datasource import Datasource, ReadTask
+
+logger = logging.getLogger(__name__)
+
+
+class TorchDatasource(Datasource[T]):
```

Comment:
Not sure where to place this class in the documentation (if at all). From what I can tell, other datasources aren't listed in the API reference. Would appreciate thoughts on this.